scripts/ops/transferOwnership.ts

```diff
@@ -17,17 +17,18 @@ async function main(): Promise<void> {
   const balance = await deployer.getBalance();
   console.log(`Account balance: ${balance.toString()}`);
 
-  const rewarderAddress = "0x78EdEeFdF8c3ad827228d07018578E89Cf159Df1"
+  const contractAddress = "0x3838956710bcc9D122Dd23863a0549ca8D5675D6" //Masterchef V2
```

Comment:
can you pull this from constants
also the chain address here is different than compared to the one on constants.ts